### DISCUSSION TITLE
Lack of documentation and training material

### DISCUSSION THREAD

**Discussion by seaj1one** [webstudio-is/webstudio-community#63] [General]
One of the biggest problems is the lack of documentation aimed at beginners. It doesn't mean people who are new to web design but new to using a tool. Webflow is doing a good and something that many tools can benefit from. 

One main thing to consider if that you can have the best tool on the market but if few know how to use it it might as well be a brick. 

If possible, try to put a big effect into documentation and video content aimed at beginners. I'd suggest videos can be based on a use case .i.e build a hero section, footer, benefit section or a card. Something that people need to build websites, at the same time you can teach people how to use your platform.

**Comment by kof** [comment]
have you seen university we are building https://www.youtube.com/@getwebstudio ? and https://webstudio.is/blog ?

Just wondering if this comes from not seeing it or because these lack certain topics.

**Reply by kof** [reply]
To speed up the conversation, if you didn't see these, I wonder where did you expect to find them. We have links in multiple places, but I wonder how to help discover them better.

**Reply by xstortionist** [reply]
example of what I'm looking for right now is documentation on grids, I assume it's simply "blocks" however it's not a common to use the term block in my experience. :)

When visiting the youtube channel I did a simple search for the term "grid" and was unfortunately presented with:
This channel has no content that matched "grid."

I just recently got the project up and running locally in WSL so I'm only about 15 minutes into building a landing page from scratch. I assume I'll catch myself up to speed through trial and error.

**Reply by ghost** [reply]
The reason you can't find information on grids is because we simply haven't added that feature yet. It's coming soon!

**Reply by xstortionist** [reply]
I figured it'd be a simple answer. Open shut case. I've now been sentenced to a life of being meme'd.

* Blocks seems to do the trick just fine.

Appreciate the quick response :)

**Comment by xstortionist** [comment]
I would agree that finding resourcing for the application is a bit clumsy and cumbersome, especially with us folk who have ADHD. However, after a bit of searching I did happen to stumble upon this issue thankfully, so now I'm headed in the right direction. :)

**Reply by ghost** [reply]
We know the learning experience isn't great right now and that's becuase we are still in beta, working fast and making lots of updates, so any documentation that we write is going to get outdated really quickly.

It's a ways away, but you'll like our documentation plan. Not only will we have nice searchable written docs, but our AI will be able to answer your questions and help you learn, like having a teacher always available! And it will be able to help with not just Webstudio, but general web/HTML/CSS knowledge too.

In the mean time, I hope you find our tooltips helpful. Hover a label or a control without a label in the style or settings panel and you'll see the CSS name and a description of the property/value. They are a work in progress but I hope they help.

**Reply by xstortionist** [reply]
To be fair, I probably won't even look at the documentation once it's released. Same thing I do when ordering furniture... don't need instructions, I'll tape it together if need be.

I consider myself a CSS sculpture. Im'a build the badest assed landing page, just wait and see, im'a show you.

**Reply by ghost** [reply]
Please do! Can't wait to see it!

Most people don't look at documentation, which is why it'll be integrated with our AI so you can just ask questions without leaving the Builder UI and get answers you need.

**Reply by kof** [reply]
documentation needs to be findable in the place where you need the information, we need to link  the stuff from tooltips and we need to add a feature for asking questions the AI

**Comment by seaj1one** [comment]
It'll be good to be different than the rest. I have seen your documentation and video's. As you add features, will you be the same as other companies and create content as an afterthought?

 I mention webflow because they understand that their product is quite technical for the end user. 
 
 Creating content based on a use case can show us the benefits instead of just showing off the features. 
 
 A big benefit is that I can use my site on different headless CMS's. I and many may not know how. In that case there's no benefit to me and many potential customers.
